test/test_any_automated_install.py

```diff
@@ -379,89 +268,28 @@ def test_FTL_detect_armv8a_no_errors(host):
-    assert expected_stdout in detectPlatform.stdout
-    expected_stdout = tick_box + " Detected ARMv7 (or newer) architecture (armv8a)"
-    assert expected_stdout in detectPlatform.stdout
-    expected_stdout = tick_box + " Downloading and Installing FTL"
-    assert expected_stdout in detectPlatform.stdout
-
-
-def test_FTL_detect_x86_64_no_errors(host):
-    """
-    confirms only x86_64 package is downloaded for FTL engine
-    """
-    detectPlatform = host.run(
-        """
-    source /opt/pihole/basic-install.sh
-    create_pihole_user
-    funcOutput=$(get_binary_name)
-    binary="pihole-FTL${funcOutput##*pihole-FTL}"
-    theRest="${funcOutput%pihole-FTL*}"
-    FTLdetect "${binary}" "${theRest}"
-    """
-    )
-    expected_stdout = info_box + " FTL Checks..."
-    assert expected_stdout in detectPlatform.stdout
-    expected_stdout = tick_box + " Detected x86_64 architecture"
-    assert expected_stdout in detectPlatform.stdout
-    expected_stdout = tick_box + " Downloading and Installing FTL"
-    assert expected_stdout in detectPlatform.stdout
-
-
-def test_FTL_detect_unknown_no_errors(host):
-    """confirms only generic package is downloaded for FTL engine"""
-    # mock uname to return generic platform
-    mock_command("uname", {"-m": ("mips", "0")}, host)
-    detectPlatform = host.run(
-        """
-    source /opt/pihole/basic-install.sh
-    create_pihole_user
-    funcOutput=$(get_binary_name)
-    binary="pihole-FTL${funcOutput##*pihole-FTL}"
-    theRest="${funcOutput%pihole-FTL*}"
-    FTLdetect "${binary}" "${theRest}"
-    """
-    )
-    expected_stdout = "Not able to detect architecture (unknown: mips)"
-    assert expected_stdout in detectPlatform.stdout
-
-
-def test_FTL_download_aarch64_no_errors(host):
-    """
-    confirms only aarch64 package is downloaded for FTL engine
-    """
-    # mock dialog answers and ensure installer dependencies
-    mock_command("dialog", {"*": ("", "0")}, host)
-    host.run(
-        """
-    source /opt/pihole/basic-install.sh
-    package_manager_detect
-    install_dependent_packages ${INSTALLER_DEPS[@]}
-    """
-    )
-    download_binary = host.run(
-        """
-    source /opt/pihole/basic-install.sh
-    create_pihole_user
-    FTLinstall "pihole-FTL-aarch64-linux-gnu"
-    """
-    )
-    expected_stdout = tick_box + " Downloading and Installing FTL"
-    assert expected_stdout in download_binary.stdout
-    assert "error" not in download_binary.stdout.lower()
+    if supported:
+        expected_stdout = info_box + " FTL Checks..."
+        assert expected_stdout in detectPlatform.stdout
+        expected_stdout = tick_box + " Detected " + detected_string + " architecture"
+        assert expected_stdout in detectPlatform.stdout
+        expected_stdout = tick_box + " Downloading and Installing FTL"
+        assert expected_stdout in detectPlatform.stdout
+    else:
+        expected_stdout = "Not able to detect architecture (unknown: " + detected_string + ")"
```

Comment:
After your last FP, the line length was changed and now tests are failing:
- https://github.com/pi-hole/pi-hole/actions/runs/7441027184/job/20242687467?pr=5542

```suggestion
        expected_stdout = (
            "Not able to detect architecture (unknown: " + detected_string + ")"
        )
```